aspen/simplates/simplate.py

```diff
@@ -164,7 +164,11 @@ def render(self, accept, context):
         context.update(self.pages[0])
         # use this as the context to execute the second page in
         exec(self.pages[1], context)
+        # refetch output, this allows the second page to override it
         output = context['state']['output']
+        # skip rendering if the second page has already filled output.body
+        if output.body:
```

Comment:
We should probably use `output.body is None` here, so that the second page can force an empty body (`''`).